<?xml version="1.0" ?>
<launch>
  <!-- params -->
  <param name="forest_file" value="$(find leg_tracker)/config/trained_leg_detector_res=0.35_outside4.yaml" />
  <param name="scan_topic" value="/scan" />
  <param name="fixed_frame" value="/laser"/>
  <param name="scan_frequency" value="10"/>  
  
  
  <!-- run detect_leg_clusters  -->
  <node pkg="leg_tracker" type="detect_leg_clusters" name="detect_leg_clusters" output="screen"/>  

  <!-- run joint_leg_tracker -->
  <node pkg="leg_tracker" type="joint_leg_tracker.py" name="joint_leg_tracker" output="screen"/> 

  <!-- run YoloToLegArray 
  <node pkg="leg_tracker" type="YoloToLegArray.py" name="YoloToLegArray" output="screen"/> -->

  <!-- run joint_leg_tracker 
  <node pkg="leg_tracker" type="yolo_tracker.py" name="yolo_tracker" output="screen"/>-->

  <!-- run joint_leg_tracker  -->
  <node pkg="leg_tracker" type="sensor_integrate.py" name="sensor_integrate" output="screen"/> 

  <!-- run joint_leg_tracker 
  <node pkg="leg_tracker" type="people_tracker.py" name="people_tracker" output="screen"/>-->


 
   <!-- run local_occupancy_grid_mapping -->
  <node pkg="leg_tracker" type="local_occupancy_grid_mapping" name="local_occupancy_grid_mapping" output="screen"/> 
   
</launch>

